### ISSUE TITLE
Can you please make it compatible with this ControlNet, which is trained using InstantX.

### ISSUE THREAD

**Issue by starinskycc** [EeroHeikkinen/ComfyUI-eesahesNodes#16]
https://huggingface.co/Shakker-Labs/FLUX.1-dev-ControlNet-Union-Pro/tree/main

**Comment by brayevalerien** [comment]
It's working properly for me with the pro version.

**Comment by alex-mitov** [comment]
> It's working properly for me with the pro version.

Can you show some examples? For me and others too, Union Pro gives produces artifacts with controlnet strengths of over ~0.5. Only the alpha version of Union works correctly for me.

**Comment by starinskycc** [comment]
https://huggingface.co/Shakker-Labs/FLUX.1-dev-ControlNet-Union-Pro/discussions/1#66d0399559df2950a5d6f71b
this one.

**Comment by brayevalerien** [comment]
> > It's working properly for me with the pro version.
> 
> Can you show some examples? For me and others too, Union Pro gives produces artifacts with controlnet strengths of over ~0.5. Only the alpha version of Union works correctly for me.

Yes, I've found the optimal strength to be bellow 0.45. Adjusting the stop at parameter can help a little bit too, but it does not seem to have much effect.

**Comment by starinskycc** [comment]
![image](https://github.com/user-attachments/assets/a766131c-5d41-4bc6-a87c-1207d8dd8f44)
![image](https://github.com/user-attachments/assets/7547fcc0-3f33-4708-a771-0e28218e918c)
